db/schema.rb

```diff
@@ -322,7 +322,7 @@
     t.string "trs_first_name", limit: 80
     t.string "trs_last_name", limit: 80
     t.date "date_of_birth"
-    t.string "trs_induction_status", limit: 16
+    t.string "trs_induction_status", limit: 18
```

Comment:
```suggestion
    t.string "trs_induction_status", limit: nil
```

Reply:
Cheers @peteryates I'll get this merged tomorrow 👍🏼